projects/ids-enterprise-ng/src/lib/stepprocess/soho-stepprocess.component.ts

```diff
@@ -82,6 +82,8 @@ export class SohoStepListItemComponent {
     }
   }
 
+  @Input() stepId?: string;
```

Comment:
why was this added? please remove it if it is only needed for the demo component